volatility/framework/plugins/windows/malfind.py

```diff
@@ -62,6 +66,43 @@ def is_vad_empty(cls, proc_layer, vad):
 
         return True
 
+    @classmethod
+    def vad_dump(cls, context: interfaces.context.ContextInterface, proc: interfaces.objects.ObjectInterface, vad)\
```

Comment:
So, in #290 there's already a method called `vad_dump` which seems like it does the same thing (although this appears to cleaner).  In either circumstance, I feel like the functionality should live in the vadinfo plugin (because there's nothing inherently malware related in the `vad_dump` method).  So I'd like to move this code into the vad plugin, but rather than messing with #290, it might be better to wait until it lands, then swap this method in for `vad_dump` in the `vadinfo` plugin, and just add a VersionRequirement on `vadinfo` and call the `vad_dump` method from there...  Hopefully #290 will land quite soon, it seems like the review's gone well.